core/backend/src/Element.ts

```diff
@@ -418,6 +418,8 @@ export abstract class GeometricElement extends Element implements GeometricEleme
   public category: Id64String;
   /** The GeometryStream for this GeometricElement. */
   public geom?: GeometryStreamProps;
+  /** How to build a GeometryStream. This is not a persistent property. It may be specified as an alternative to `geom` when inserting or updating an element.  */
+  public elementGeometryBuilderParams?: ElementGeometryUpdate;
```

Comment:
Mark alpha

Reply:
Done

Reply:
Did you push that change? It is still missing a release tag.

Reply:
I missed that case. Thanks for pointing it out.